lua-resty-websocket-kong-0.1.0-1.rockspec

```diff
@@ -0,0 +1,24 @@
+package = "lua-resty-websocket-kong"
+version = "0.1.0"
```

Comment:
```suggestion
version = "0.1.0-1"
```

The LuaRocks version needs a versioned rockspec as well, by convention it starts at `1`.